spark/Dockerfile

```diff
@@ -77,6 +75,7 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
 
 # Add iceberg spark runtime jar to IJava classpath
 ENV IJAVA_CLASSPATH=/opt/spark/jars/*
+ENV JAVA_PATH="/opt/spark/jars/*"
```

Comment:
is this used anywhere?

Reply:
Not sure, was a while ago. I've removed it